spec/index.bs

```diff
@@ -2886,6 +2886,7 @@ enum GPUFeatureName {
     "clip-distances",
     "dual-source-blending",
     "subgroups",
+    "storage-texture-format-tier1",
```

Comment:
nit: "Tier 1" made sense when WebGPU didn't have any RW storage texture formats in Core. But now that Core has some, I would consider that "Tier 1" and still call the extension "Tier 2" like was originally proposed for this featureset.

Reply:
Done